models/classes/LtiProvider/LtiProviderService.php

```diff
@@ -100,7 +100,7 @@ static function ($providers, LtiProviderRepositoryInterface $implementation) use
     private function aggregate($result, $method)
     {
         foreach ($this->getOption(self::LTI_PROVIDER_LIST_IMPLEMENTATIONS) as $implementation) {
-            $this->propagate($implementation);
+            $implementation = $this->buildService($implementation);
```

Comment:
it seems like `\oat\oatbox\service\ServiceManager::propagate` which will be called from `buildService` method in this particular case differs from `\oat\oatbox\service\ServiceManagerAwareTrait::propagate` which was called before in a logger part. 

**suggestion:** Probably it is not a bad idea to leave propagate and add `buildService` call before it.

Reply:
Thank you, I've updated the implementation to make sure logger is propagated as well 👍🏻 And added a test case